pkg/search/postgres/common.go

```diff
@@ -623,7 +599,21 @@ func RunSearchRequestForSchema(ctx context.Context, schema *walker.Schema, q *v1
 	for i, field := range extraSelectedFields {
 		bufferToScanRowInto[i+len(query.SelectedFields)+numFieldsForPrimaryKey] = mustAllocForDataType(field.FieldType)
 	}
+
 	recordIDIdxMap := make(map[string]int)
+	var searchResults []searchPkg.Result
+
+	rows, err := tracedQuery(ctx, db, queryStr, query.Data...)
+	if err != nil {
+		if !buildinfo.ReleaseBuild {
+			debug.PrintStack()
+			log.Errorf("Query issue: %s %+v: %v", queryStr, query.Data, err)
+		}
+		return nil, err
+	}
+	defer rows.Close()
+	log.Debugf("SEARCH: ran query %s; data %+v", queryStr, query.Data)
```

Comment:
I think I probably want to remove this debug log or at least the query.Data portion because this could log secrets

Reply:
Add a `TODO` in some form?

Reply:
I created a jira ticket for this and marked it as a blocker for Postgres

Reply:
done